docker-compose.yml

```diff
@@ -40,7 +40,7 @@ services:
   web:
     extends:
       service: service_base
-    command: gunicorn -b 0.0.0.0:5000 -t 3600 --access-logfile "-" inspirehep.wsgi
+    command: gunicorn -b 0.0.0.0:5000 -t 3600 --access-logfile "-" inspirehep.wsgi_with_coverage
```

Comment:
This is only for development and nightly builds. The file you are looking for is `docker-compose.test.yml`